tasking/pulp/tasking/tasks.py

```diff
@@ -204,17 +205,23 @@ def apply_async_with_reservation(self, resource_type, resource_id, *args, **kwar
                 task_status.tags.create(name=tag)
 
         # Call the outer task which is a promise to call the real task when it can.
-        _queue_reserved_task.apply_async(args=[task_name, inner_task_id, resource_id, args, kwargs],
-                                         queue=RESOURCE_MANAGER_QUEUE)
+        _queue_reserved_task.apply_async(args=[task_name, inner_task_id, resource_id, async_args,
+                                               async_kwargs], queue=RESOURCE_MANAGER_QUEUE)
         return AsyncResult(inner_task_id)
 
-    def apply_async(self, *args, **kwargs):
+    def apply_async(self, *celery_args, tags=[], group_id=None, **celery_kwargs):
```

Comment:
I think this also needs to be `celery_args=None` instead of `*celery_args` and `celery_kwargs=None` instead of `**celery_kwargs`.